example/lib/place_source.dart

```diff
@@ -68,16 +69,24 @@ class PlaceSymbolBodyState extends State<PlaceSymbolBody> {
   }
 
   Future<void> addLayer(String imageLayerId, String imageSourceId) {
+    if(layerAdded) {
+      removeLayer(imageLayerId);
+    }
+    layerAdded = true;
```

Comment:
might it be a good idea to use setState? Even tho currently nothing in the ui depends on it, still feels like a good practice so set state only with setSate method 😉

Reply:
Thanks for your advice. Applied setState for it.